src/pug/head.pug

```diff
@@ -3,3 +3,4 @@ meta(name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit
 base(href=`${ release === 'insights' ? '/' : '/beta/'}`)
 link(rel='stylesheet', href='https://use.fontawesome.com/releases/v5.0.10/css/all.css', integrity='sha384-+d0P83n9kaQMCwj8F4RJB66tzIwOKmrdb46+porD/OvrJ+37WqIM7UoBtwHO6Nlg', crossorigin='anonymous')
 link(rel='stylesheet', href='./apps/chrome/chrome.css')
+script(id="adobe_dtm" src="//www.cloud.redhat.com" type="text/javascript")
```

Comment:
Lol wat dis?
src="//www.cloud.redhat.com" ? That aint no script. Its not even a valid URL.

Reply:
From the mojo:

```
<script id="adobe_dtm" src="{DPAL URL}" type="text/javascript"></script>

DPAL Production URLs:
//www.redhat.com/dtm.js - Sends data to the "Red Hat Catchall" report suite in Adobe Analytics
``

Reply:
Oh nevermind. I didn't push up the change for this

Reply:
@iphands Unstaged commits. I was debugging to see if I could get around the catchall

Reply:
@iphands updated